view/lang/en-us/strings.php

```diff
@@ -778,6 +778,10 @@ function string_plural_select_en_us($n){
 $a->strings["Database"] = "Database";
 $a->strings["DB updates"] = "DB updates";
 $a->strings["Inspect Queue"] = "Inspect queue";
+$a->strings["Inspect Worker Queue"] = "Inspect worker queue";
```

Comment:
You don't have to include this translation. It should be made once the master messages.po file is picked up at Transifex. Otherwise probability is hight that your translation will get lost.